myproject/api/serializers.py

```diff
@@ -1,6 +1,11 @@
 from rest_framework import serializers
+from base.models import Lists
 from base.models import User
 
+class ListsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Lists
```

Comment:
you need to readd the field all thing that i think got dropped when u did the merge with main, it was erroring on me without it
```python
class ListsSerializer(serializers.ModelSerializer):
    class Meta:
        model = Lists
        fields = '__all__'
  ```

Reply:
Sorry for the missing line added it so all fields are read